src/app/vm-logs/redux/vm-logs.actions.ts

```diff
@@ -28,12 +34,25 @@ export class LoadVmLogsResponse implements Action {
 
 }
 
-export class VmLogsFilterUpdate implements Action {
-  type = VM_LOGS_FILTER_UPDATE;
+export class LoadVmLogFilesRequest implements Action {
+  type = LOAD_VM_LOG_FILES_REQUEST;
 
-  constructor(public payload: any) {
+  constructor(public payload?: never) {
```

Comment:
readonly (like everywhere)

Reply:
I fixed `any`'s here: https://github.com/bwsw/cloudstack-ui/pull/1346/files. I will update these `any`'s when merging, because type check will fail if I update only some of these.